src/components/confetti/Confetti.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { Confetti } from "./Confetti";
+import React from "react";
+
+const meta: Meta<typeof Confetti> = {
+  component: Confetti,
+  parameters: {
+    chromatic: { disableSnapshot: true },
+  },
+  decorators: [
+    (StoryFn) => (
+      <div style={{ height: "100vh", width: "100vw" }}>
+        <button>I am clickable</button>
```

Comment:
this made me confused as it doesn't do anything, but I thought you could toggle showing confetti on click 😆

Reply:
It was just about showcasing that the confetti is on top of the contents, but the content below is still clickable.